content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -1,74 +1,355 @@
+
+Should the :guilabel:`Commit Transactions` checkbox be ticked, then, the transactions in the Odoo
+database will be committed for reporting in AvaTax.
 
 Address validation
-==================
+------------------
 
-Manually validate customer addresses by clicking the :guilabel:`Validate address` link in the
-customer form view.
+.. important::
+   The :guilabel:`Address validation` feature only works with partners in North America.
 
-.. image:: avatax/validate-customer-address.png
-   :align: center
-   :alt: Validate customer addresses
+Additionally, tick the checkbox next to the :guilabel:`Address validation` field.
 
-If preferred, choose to keep the newly validated address or the original address in the wizard that
-pops up.
+.. important::
+   For accurate tax calculations it is best practice to enter a complete address for the contacts
+   saved in the database. However, AvaTax can still function by implementing a best effort attempt
+   using only the :guilabel:`Country`, :guilabel:`State`, and :guilabel:`Zip code`. These are the
+   three minimum required fields.
 
-.. image:: avatax/choose-customer-address.png
-   :align: center
-   :alt: Address validation wizard
+:guilabel:`Save` the settings to implement the configuration.
+
+.. tip::
+   Manually :guilabel:`Validate` the address by navigating to :menuselection:`Contacts app` and
+   selecting a contact. Now that the AvaTax module have been configured on the database, a
+   :guilabel:`Validate` button appears directly below the :guilabel:`Address`.
+
+   Click :guilabel:`Validate` and a pop-up window appears with a :guilabel:`Validated Address` and
+   :guilabel:`Original Address` listed. If the :guilabel:`Validated Address` is the correct mailing
+   address for tax purposes click :guilabel:`Save Validated`.
 
-.. _avatax/tax-calculation:
+   .. image:: avatax/validate-address.png
+      :align: center
+      :alt: Validate address pop-up window in Odoo with save validated button and validated address
+            highlighted.
 
-Tax calculation
+.. warning::
+   All previously entered addresses for contacts in the Odoo database will need to be validated
+   using the manually validate process outlined above. Addresses are not automatically validated if
+   they were entered previously. This only occurs upon tax calculation.
+
+Sync parameters
+---------------
+
+Upon finishing the configuration and settings of the AvaTax section click the :guilabel:`Sync
+Parameters`. This action will synchronize the exemption codes from AvaTax.
+
+.. _avatax/fiscal_positions:
+
+Fiscal position
 ===============
 
-Automatically calculate taxes on Odoo quotations and invoices with Avatax by confirming the
-documents. Alternatively, calculate the taxes manually by clicking the :guilabel:`Compute taxes
-using Avatax` button while these documents are in draft mode.
+Next, navigate to :menuselection:`Configuration --> Accounting --> Fiscal Positions`. A new
```

Comment:
Please verify with Lara, but I think we use `Accounting: Fiscal Positions` when Fiscal Positions is the button to click and `Accounting` is the section title of the menu option.
```suggestion
Next, navigate to :menuselection:`Accounting app --> Configuration --> Accounting: Fiscal Positions`. A new
```
![image](https://github.com/odoo/documentation/assets/23611711/7a292dbe-1421-4168-8c56-e80157248825)